tests/test_mcp_clients.py

```
@@ -634,3 +634,204 @@ async def get_items(count: int):
+    tool_calls = [
+        {"code": "import math", "request_id": "parity"},
+        {"code": "result = math.factorial(10)", "request_id": "parity"},
+        {"code": "print(result)", "request_id": "parity"},
+        {"code": "x = [i**2 for i in range(5)]", "request_id": "parity"},
+        {"code": "print(sum(x))", "request_id": "parity"},
+    ]
+
+    direct_results = await _run_tool_sequence(direct, tool_calls)
+    mcp_results = await _run_tool_sequence(mcp_tool, tool_calls)
+
+    for i, (d, m) in enumerate(zip(direct_results, mcp_results)):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)